interpreter/shared/src/test/scala/sigma/ast/SigmaBuilderTest.scala

```diff
@@ -122,6 +122,13 @@ class SigmaBuilderTest extends AnyPropSpec with ScalaCheckPropertyChecks with Ma
     testSuccess(arr, TransformingSigmaBuilder.mkCollectionConstant(arr, c.tpe))
   }
 
+  def testArrayFailure[T <: SType]
+    (v: T#WrappedType, c: Constant[T])(implicit t: RType[T#WrappedType]) = {
+    // for any Byte and Short value `v`, lifting of array should succeed
```

Comment:
Isn't is generic enough to work for any type? Comment somewhat misleading.

Reply:
fixed